dashboard/app/views/home/_survey_diversity.haml

```diff
@@ -1,91 +1,82 @@
-- if show_survey?
-  - survey_url = "https://docs.google.com/forms/d/10Qebl7Gi9DTf-RDt-0v78jVFdzJN0mWTSHgAuk6V5gU/viewform?entry.692842963=__other_option__&entry.692842963.other_option_response=#{current_user.id}&entry.1146050975&entry.796977513&entry.999252886"
-  .survey{style: "border: 4px solid #a69bc1; border-radius: 15px; padding:40px"}
-    #surveythanks{style: "display: none; font-size: 14px"}
-      Thank you for telling us about your students!
-      %br/
-      %br/
-      We're making changes to improve Code Studio. We'd love your feedback on how we can make it work best for your classroom.
-      %br/
-      %br/
-      %a{href: survey_url, target: "_blank"}
-        %button.btn-primary
-          Go to survey
+- survey_url = "https://docs.google.com/a/code.org/forms/d/e/1FAIpQLSd7MppqXJi4FpURELDH3RyMG6PgevBVPaH373K0h-FSI_3FUg/viewform"
```

Comment:
It seems odd (bad UX) that there is exactly one required question in this survey.